nasa-gibs/worldview
# Add standard reference links

## Description

Fixes WV-2861

Added references to standard layers for OMPS layers

## How To Test

1. `npm run build && npm start`
2. Click on Add Layers, type "OMPS"
3. Verify that there are two sets of links under "References" one for NRT and one for the Standard dataset landing pages.

@nasa-gibs/worldview
